fileattrs/elfoptimized.attr

```diff
@@ -0,0 +1,4 @@
+%__elf_requires		%{_rpmconfigdir}/elfdeps --requires %{?__filter_GLIBC_PRIVATE:--filter-private}
```

Comment:
Do the macros need to use the `__elfoptimized` prefix instead of `__elf`? I've noticed that the macros from other .attr files are named after the file itself (without suffix).

Reply:
Ugh, sorry. Bad copy & paste.